### PR TITLE
feat: introduce mainnet profile

### PR DESCRIPTION
**Pull request by mrekucci**
## Describe your changes

Introduces mainnet profile.

## Checklist before requesting a review

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2 [hunk 1]
```diff
@@ -148,8 +148,10 @@ job "{{ job.name }}" {
             --directory /usr/local \
             --strip-components=1
 
+          find local/contracts -type f -name "._*.sol" -exec rm -f {} +
```
Comment (shaspitz) [inline infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2:151]: Odd that we need this line, but it shouldn't hurt anything

**Review by shaspitz — APPROVED** [review]
LGTM, also see https://github.com/primev/mev-commit/pull/545

**Comment by shaspitz** [comment]
I'll set this PR to close https://github.com/primev/mev-commit/issues/338

File: infrastructure/nomad/playbooks/variables/profiles.yml [hunk 2]
```diff
@@ -737,6 +737,21 @@ profiles:
...
+    jobs:
+      - *artifacts_job
+      - *datadog_agent_logs_collector_job
+      - *otel_collector_job
+      - *mev_commit_geth_bootnode1_job
+      - *mev_commit_geth_signer_node1_job
+      - *mev_commit_geth_member_node_job
+      - *contracts_deployer_job
+      - *mev_commit_bridge_job
+      - *mev_commit_dashboard_job
```
Comment (aloknerurkar) [inline infrastructure/nomad/playbooks/variables/profiles.yml:750]: I think I forgot to add a rule in our nginx for the dashboard. We can call it dashboard.mev-commit.xyz maybe. It uses a different port. If possible add this as well.

**Review by aloknerurkar — APPROVED** [review]

**Pull request merged**